specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -893,6 +955,23 @@
         }
       }
     },
+    "ComposeRequest": {
+      "description": "Request contract for compose operation.",
+      "required": [
+        "modelIds"
+      ],
+      "type": "object",
+      "properties": {
+        "modelIds": {
+          "description": "List of model ids to compose.",
+          "uniqueItems": false,
```

Comment:
"uniqueItems": do we allow duplicated modelIds?

Reply:
We don't use them, and when we do the validation on FE side, we look at the number of unique models. But it doesn't seem necessary to enforce uniqueness here, I think we can be tolerant to this type of user errors.. Let me know if you still think we need a "true" here.

Reply:
we could ship this 'as is',  in preview,  and collect feedback.  so no need to change now. thanks.

Reply:
also, when the compose model list has a composed model,  it's super hard to ensure the sub-models won't duplicate with any models.   so leaving this uniqueItems as false makes sense.